doc/whats_new/v1.4.rst

```diff
@@ -237,6 +237,8 @@ Changelog
 :mod:`sklearn.compose`
 ......................
 
+- |API| outputs that use pandas extension dtypes and contain `pd.NA` in :class:`~compose.ColumnTransformer` now result in a `FutureWarning` and will cause a `ValueError` in version 1.6, unless the output container has been configured as "pandas" with `set_output(transform="pandas")`. Before, such outputs resulted in numpy arrays of dtype `object` containing `pd.NA` which could not be converted to numpy floats and caused errors when passed to other scikit-learn estimators. :pr:`27734` by :user:`Jérôme Dockès <jeromedockes>`.
```

Comment:
Could you please rewrap to ~80 columns to be consistent with the style of the rest of the changelog?